include/spdk/bdev.h

```diff
@@ -229,6 +230,8 @@ struct spdk_bdev_io *spdk_bdev_unmap(struct spdk_bdev *bdev, struct spdk_io_chan
 struct spdk_bdev_io *spdk_bdev_flush(struct spdk_bdev *bdev, struct spdk_io_channel *ch,
 				     uint64_t offset, uint64_t length,
 				     spdk_bdev_io_completion_cb cb, void *cb_arg);
+struct spdk_bdev_io *spdk_bdev_passthru(struct spdk_bdev *bdev, void *cmd, void *buf,
+					uint16_t nbytes, spdk_bdev_io_completion_cb cb, void *cb_arg);
```

Comment:
Any particular reason nbytes is a uint16_t here?  I think a size_t would be better as a buffer size.